src/software/thunderscope/thunderscope_config.py

```diff
@@ -487,7 +490,10 @@ def configure_replay_view(
                     replay_log=blue_replay_log,
                     visualization_buffer_size=visualization_buffer_size,
                     extra_widgets=[],
+                    frame_swap_counter=frame_swap_frametime_counter,
+                    refresh_counter=refresh_func_frametime_counter,
```

Comment:
You should instantiate the FrametimeCounters right here instead of using the same variables for both the blue_replay_log and yellow_replay_log branches, since I think you might be able to run thunderscope with both a blue and yellow replay at the same time

Reply:
Wait, this is a thing. Ok will do